auth/utils/index.ts

```diff
@@ -14,6 +14,23 @@
  * limitations under the License.
  */
 
+/**
+ * @module utils
+ * Helper functions used by generated SDKs.
+ *
+ * Identity and Access Management (IAM)
+ * Cloud Pak for Data
+ *
+ * The token managers sit inside of an authenticator and do the work to retrieve
+ * tokens where as the authenticators add these tokens to the actual request.
+ *
+ * functions:
+ *   getAuthenticatorFromEnvironment: Get authenticator from external sources.
+ *   readExternalSources: Get config object from external sources.
+ */
+
+
+
 export * from './helpers';
 export * from './read-credentials-file';
```

Comment:
I think we really only want to expose readExternalSources and not the lower-level functions that it calls.

Reply:
I think that's all that is really being exposed here. @dpopp07 is this true?

Reply:
Yes, that's true. We're exporting all exported methods in `read-credentials-file` but only one method is exported from there - `readExternalSources`